suricata/update/commands/updatesources.py

```diff
@@ -27,11 +28,36 @@
 
 logger = logging.getLogger()
 
+
 def register(parser):
     parser.set_defaults(func=update_sources)
 
+
+def compare_sources(initial_content, final_content):
+    initial_sources = initial_content.get("sources")
+    final_sources = final_content.get("sources")
+    added_sources = {source: final_sources[source]
+                     for source in final_sources if source not in initial_sources}
+    removed_sources = {source: initial_sources[source]
+                       for source in initial_sources if source not in final_sources}
+    if initial_content == final_content:
```

Comment:
Also, we can move line 43-45 on line 37.

This is initial check with the arguments itself on the equality. This way, we can avoid unnecessary computation, in case the dict of `initial_content` and `final_content` is long enough.

Thoughts?

Reply:
I do not think it is going to be very long anytime but doesn't hurt to avoid extra computations. Taking this into account too.